### ISSUE TITLE
basic example of angular2-data-table with angular-cli does not work

### ISSUE THREAD

**Issue by ipassynk** [swimlane/angular2-data-table#175]
**I'm submitting a ...**  (check one with "x")
```
[x ] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => Please do not submit support request here
```

**Current behavior**

a. Install angular-cli
ng version:
angular-cli: 1.0.0-beta.16
node: 4.5.0
os: darwin x64

b. npm install --save angular2-data-table
package.json
dependencies:
{
 "angular2-data-table": "^0.9.0",
}

c. in generated app.module.ts add
import { Angular2DataTableModule } from 'angular2-data-table';

and add Angular2DataTableModule to list of imports

d.Add from basic example code into app.component.ts

``` 
import { Component } from '@angular/core';
import {ColumnMode, TableColumn, TableOptions} from "angular2-data-table";

@Component({
  selector: 'app-root',
  templateUrl: './app.component.html',
  styleUrls: ['./app.component.css']
})
export class AppComponent {
  rows = [];

  options = new TableOptions({
    columnMode: ColumnMode.force,
    headerHeight: 50,
    footerHeight: 50,
    rowHeight: 'auto',
    columns: [
      new TableColumn({prop: 'name'}),
      new TableColumn({name: 'Gender'}),
      new TableColumn({name: 'Company'})
    ]
  });

  constructor() {
    let data = [
      {name:'julia', 'Gender':'F', Company: 'My company'}
    ];
    this.rows.push(...data);
  }
}
```
e. Add html from basic example to app.component.html
```
<div>
  <h3>basic</h3>
  <datatable
    class='material striped'
    [rows]='rows'
    [options]='options'>
  </datatable>
</div>
```
f. Type: ng.serve - this is no error during the compilation

g: Open browser http://localhost:4200/
and see the following errors on the console:

error_handler.js:45EXCEPTION: Error in ./DataTableHeader class DataTableHeader - inline template:29:8 caused by: Cannot read property 'name' of undefinedErrorHandler.handleError @ error_handler.js:45(anonymous function) @ application_ref.js:209ZoneDelegate.invoke @ zone.js:203onInvoke @ ng_zone_impl.js:43ZoneDelegate.invoke @ zone.js:202Zone.run @ zone.js:96(anonymous function) @ zone.js:462ZoneDelegate.invokeTask @ zone.js:236onInvokeTask @ ng_zone_impl.js:34ZoneDelegate.invokeTask @ zone.js:235Zone.runTask @ zone.js:136drainMicroTaskQueue @ zone.js:368Promise.resolve (async)scheduleQueueDrain @ zone.js:341scheduleMicroTask @ zone.js:349ZoneDelegate.scheduleTask @ zone.js:219Zone.scheduleMicroTask @ zone.js:150scheduleResolveOrReject @ zone.js:460ZoneAwarePromise.then @ zone.js:536RuntimeCompiler._compileComponents @ runtime_compiler.js:157RuntimeCompiler._compileModuleAndComponents @ runtime_compiler.js:64RuntimeCompiler.compileModuleAsync @ runtime_compiler.js:55PlatformRef_._bootstrapModuleWithZone @ application_ref.js:303PlatformRef_.bootstrapModule @ application_ref.js:285(anonymous function) @ main.ts:12__webpack_require__ @ bootstrap 8ad11c6…:52(anonymous function) @ zone.js:1327__webpack_require__ @ bootstrap 8ad11c6…:52webpackJsonpCallback @ bootstrap 8ad11c6…:23(anonymous function) @ main.bundle.js:1
error_handler.js:47ORIGINAL EXCEPTION: Cannot read property 'name' of undefinedErrorHandler.handleError @ error_handler.js:47(anonymous function) @ application_ref.js:209ZoneDelegate.invoke @ zone.js:203onInvoke @ ng_zone_impl.js:43ZoneDelegate.invoke @ zone.js:202Zone.run @ zone.js:96(anonymous function) @ zone.js:462ZoneDelegate.invokeTask @ zone.js:236onInvokeTask @ ng_zone_impl.js:34ZoneDelegate.invokeTask @ zone.js:235Zone.runTask @ zone.js:136drainMicroTaskQueue @ zone.js:368Promise.resolve (async)scheduleQueueDrain @ zone.js:341scheduleMicroTask @ zone.js:349ZoneDelegate.scheduleTask @ zone.js:219Zone.scheduleMicroTask @ zone.js:150scheduleResolveOrReject @ zone.js:460ZoneAwarePromise.then @ zone.js:536RuntimeCompiler._compileComponents @ runtime_compiler.js:157RuntimeCompiler._compileModuleAndComponents @ runtime_compiler.js:64RuntimeCompiler.compileModuleAsync @ runtime_compiler.js:55PlatformRef_._bootstrapModuleWithZone @ application_ref.js:303PlatformRef_.bootstrapModule @ application_ref.js:285(anonymous function) @ main.ts:12__webpack_require__ @ bootstrap 8ad11c6…:52(anonymous function) @ zone.js:1327__webpack_require__ @ bootstrap 8ad11c6…:52webpackJsonpCallback @ bootstrap 8ad11c6…:23(anonymous function) @ main.bundle.js:1
error_handler.js:50ORIGINAL STACKTRACE:ErrorHandler.handleError @ error_handler.js:50(anonymous function) @ application_ref.js:209ZoneDelegate.invoke @ zone.js:203onInvoke @ ng_zone_impl.js:43ZoneDelegate.invoke @ zone.js:202Zone.run @ zone.js:96(anonymous function) @ zone.js:462ZoneDelegate.invokeTask @ zone.js:236onInvokeTask @ ng_zone_impl.js:34ZoneDelegate.invokeTask @ zone.js:235Zone.runTask @ zone.js:136drainMicroTaskQueue @ zone.js:368Promise.resolve (async)scheduleQueueDrain @ zone.js:341scheduleMicroTask @ zone.js:349ZoneDelegate.scheduleTask @ zone.js:219Zone.scheduleMicroTask @ zone.js:150scheduleResolveOrReject @ zone.js:460ZoneAwarePromise.then @ zone.js:536RuntimeCompiler._compileComponents @ runtime_compiler.js:157RuntimeCompiler._compileModuleAndComponents @ runtime_compiler.js:64RuntimeCompiler.compileModuleAsync @ runtime_compiler.js:55PlatformRef_._bootstrapModuleWithZone @ application_ref.js:303PlatformRef_.bootstrapModule @ application_ref.js:285(anonymous function) @ main.ts:12__webpack_require__ @ bootstrap 8ad11c6…:52(anonymous function) @ zone.js:1327__webpack_require__ @ bootstrap 8ad11c6…:52webpackJsonpCallback @ bootstrap 8ad11c6…:23(anonymous function) @ main.bundle.js:1
error_handler.js:51TypeError: Cannot read property 'name' of undefined
    at DataTableHeaderCell.get [as name] (index.umd.js:1263)
    at DebugAppView._View_DataTableHeader4.detectChangesInternal (DataTableHeader.ngfactory.js:820)
    at DebugAppView.AppView.detectChanges (view.js:272)
    at DebugAppView.detectChanges (view.js:377)
    at DebugAppView.AppView.detectContentChildrenChanges (view.js:290)
    at DebugAppView._View_DataTableHeader3.detectChangesInternal (DataTableHeader.ngfactory.js:620)
    at DebugAppView.AppView.detectChanges (view.js:272)
    at DebugAppView.detectChanges (view.js:377)
    at DebugAppView.AppView.detectContentChildrenChanges (view.js:290)
    at DebugAppView._View_DataTableHeader0.detectChangesInternal (DataTableHeader.ngfactory.js:131)ErrorHandler.handleError @ error_handler.js:51(anonymous function) @ application_ref.js:209ZoneDelegate.invoke @ zone.js:203onInvoke @ ng_zone_impl.js:43ZoneDelegate.invoke @ zone.js:202Zone.run @ zone.js:96(anonymous function) @ zone.js:462ZoneDelegate.invokeTask @ zone.js:236onInvokeTask @ ng_zone_impl.js:34ZoneDelegate.invokeTask @ zone.js:235Zone.runTask @ zone.js:136drainMicroTaskQueue @ zone.js:368Promise.resolve (async)scheduleQueueDrain @ zone.js:341scheduleMicroTask @ zone.js:349ZoneDelegate.scheduleTask @ zone.js:219Zone.scheduleMicroTask @ zone.js:150scheduleResolveOrReject @ zone.js:460ZoneAwarePromise.then @ zone.js:536RuntimeCompiler._compileComponents @ runtime_compiler.js:157RuntimeCompiler._compileModuleAndComponents @ runtime_compiler.js:64RuntimeCompiler.compileModuleAsync @ runtime_compiler.js:55PlatformRef_._bootstrapModuleWithZone @ application_ref.js:303PlatformRef_.bootstrapModule @ application_ref.js:285(anonymous function) @ main.ts:12__webpack_require__ @ bootstrap 8ad11c6…:52(anonymous function) @ zone.js:1327__webpack_require__ @ bootstrap 8ad11c6…:52webpackJsonpCallback @ bootstrap 8ad11c6…:23(anonymous function) @ main.bundle.js:1
error_handler.js:54ERROR CONTEXT:ErrorHandler.handleError @ error_handler.js:54(anonymous function) @ application_ref.js:209ZoneDelegate.invoke @ zone.js:203onInvoke @ ng_zone_impl.js:43ZoneDelegate.invoke @ zone.js:202Zone.run @ zone.js:96(anonymous function) @ zone.js:462ZoneDelegate.invokeTask @ zone.js:236onInvokeTask @ ng_zone_impl.js:34ZoneDelegate.invokeTask @ zone.js:235Zone.runTask @ zone.js:136drainMicroTaskQueue @ zone.js:368Promise.resolve (async)scheduleQueueDrain @ zone.js:341scheduleMicroTask @ zone.js:349ZoneDelegate.scheduleTask @ zone.js:219Zone.scheduleMicroTask @ zone.js:150scheduleResolveOrReject @ zone.js:460ZoneAwarePromise.then @ zone.js:536RuntimeCompiler._compileComponents @ runtime_compiler.js:157RuntimeCompiler._compileModuleAndComponents @ runtime_compiler.js:64RuntimeCompiler.compileModuleAsync @ runtime_compiler.js:55PlatformRef_._bootstrapModuleWithZone @ application_ref.js:303PlatformRef_.bootstrapModule @ application_ref.js:285(anonymous function) @ main.ts:12__webpack_require__ @ bootstrap 8ad11c6…:52(anonymous function) @ zone.js:1327__webpack_require__ @ bootstrap 8ad11c6…:52webpackJsonpCallback @ bootstrap 8ad11c6…:23(anonymous function) @ main.bundle.js:1
error_handler.js:55DebugContext {_view: _View_DataTableHeader4, _nodeIndex: 0, _tplRow: 29, _tplCol: 8}ErrorHandler.handleError @ error_handler.js:55(anonymous function) @ application_ref.js:209ZoneDelegate.invoke @ zone.js:203onInvoke @ ng_zone_impl.js:43ZoneDelegate.invoke @ zone.js:202Zone.run @ zone.js:96(anonymous function) @ zone.js:462ZoneDelegate.invokeTask @ zone.js:236onInvokeTask @ ng_zone_impl.js:34ZoneDelegate.invokeTask @ zone.js:235Zone.runTask @ zone.js:136drainMicroTaskQueue @ zone.js:368Promise.resolve (async)scheduleQueueDrain @ zone.js:341scheduleMicroTask @ zone.js:349ZoneDelegate.scheduleTask @ zone.js:219Zone.scheduleMicroTask @ zone.js:150scheduleResolveOrReject @ zone.js:460ZoneAwarePromise.then @ zone.js:536RuntimeCompiler._compileComponents @ runtime_compiler.js:157RuntimeCompiler._compileModuleAndComponents @ runtime_compiler.js:64RuntimeCompiler.compileModuleAsync @ runtime_compiler.js:55PlatformRef_._bootstrapModuleWithZone @ application_ref.js:303PlatformRef_.bootstrapModule @ application_ref.js:285(anonymous function) @ main.ts:12__webpack_require__ @ bootstrap 8ad11c6…:52(anonymous function) @ zone.js:1327__webpack_require__ @ bootstrap 8ad11c6…:52webpackJsonpCallback @ bootstrap 8ad11c6…:23(anonymous function) @ main.bundle.js:1
error_handler.js:45EXCEPTION: Error in ./DataTableHeader class DataTableHeader - inline template:29:8 caused by: Cannot read property 'name' of undefinedErrorHandler.handleError @ error_handler.js:45next @ application_ref.js:273schedulerFn @ async.js:82SafeSubscriber.__tryOrUnsub @ Subscriber.js:223SafeSubscriber.next @ Subscriber.js:172Subscriber._next @ Subscriber.js:125Subscriber.next @ Subscriber.js:89Subject.next @ Subject.js:55EventEmitter.emit @ async.js:74onError @ ng_zone.js:120onHandleError @ ng_zone_impl.js:64ZoneDelegate.handleError @ zone.js:207Zone.runTask @ zone.js:139drainMicroTaskQueue @ zone.js:368Promise.resolve (async)scheduleQueueDrain @ zone.js:341scheduleMicroTask @ zone.js:349ZoneDelegate.scheduleTask @ zone.js:219Zone.scheduleMicroTask @ zone.js:150scheduleResolveOrReject @ zone.js:460ZoneAwarePromise.then @ zone.js:536RuntimeCompiler._compileComponents @ runtime_compiler.js:157RuntimeCompiler._compileModuleAndComponents @ runtime_compiler.js:64RuntimeCompiler.compileModuleAsync @ runtime_compiler.js:55PlatformRef_._bootstrapModuleWithZone @ application_ref.js:303PlatformRef_.bootstrapModule @ application_ref.js:285(anonymous function) @ main.ts:12__webpack_require__ @ bootstrap 8ad11c6…:52(anonymous function) @ zone.js:1327__webpack_require__ @ bootstrap 8ad11c6…:52webpackJsonpCallback @ bootstrap 8ad11c6…:23(anonymous function) @ main.bundle.js:1
error_handler.js:47ORIGINAL EXCEPTION: Cannot read property 'name' of undefinedErrorHandler.handleError @ error_handler.js:47next @ application_ref.js:273schedulerFn @ async.js:82SafeSubscriber.__tryOrUnsub @ Subscriber.js:223SafeSubscriber.next @ Subscriber.js:172Subscriber._next @ Subscriber.js:125Subscriber.next @ Subscriber.js:89Subject.next @ Subject.js:55EventEmitter.emit @ async.js:74onError @ ng_zone.js:120onHandleError @ ng_zone_impl.js:64ZoneDelegate.handleError @ zone.js:207Zone.runTask @ zone.js:139drainMicroTaskQueue @ zone.js:368Promise.resolve (async)scheduleQueueDrain @ zone.js:341scheduleMicroTask @ zone.js:349ZoneDelegate.scheduleTask @ zone.js:219Zone.scheduleMicroTask @ zone.js:150scheduleResolveOrReject @ zone.js:460ZoneAwarePromise.then @ zone.js:536RuntimeCompiler._compileComponents @ runtime_compiler.js:157RuntimeCompiler._compileModuleAndComponents @ runtime_compiler.js:64RuntimeCompiler.compileModuleAsync @ runtime_compiler.js:55PlatformRef_._bootstrapModuleWithZone @ application_ref.js:303PlatformRef_.bootstrapModule @ application_ref.js:285(anonymous function) @ main.ts:12__webpack_require__ @ bootstrap 8ad11c6…:52(anonymous function) @ zone.js:1327__webpack_require__ @ bootstrap 8ad11c6…:52webpackJsonpCallback @ bootstrap 8ad11c6…:23(anonymous function) @ main.bundle.js:1
error_handler.js:50ORIGINAL STACKTRACE:ErrorHandler.handleError @ error_handler.js:50next @ application_ref.js:273schedulerFn @ async.js:82SafeSubscriber.__tryOrUnsub @ Subscriber.js:223SafeSubscriber.next @ Subscriber.js:172Subscriber._next @ Subscriber.js:125Subscriber.next @ Subscriber.js:89Subject.next @ Subject.js:55EventEmitter.emit @ async.js:74onError @ ng_zone.js:120onHandleError @ ng_zone_impl.js:64ZoneDelegate.handleError @ zone.js:207Zone.runTask @ zone.js:139drainMicroTaskQueue @ zone.js:368Promise.resolve (async)scheduleQueueDrain @ zone.js:341scheduleMicroTask @ zone.js:349ZoneDelegate.scheduleTask @ zone.js:219Zone.scheduleMicroTask @ zone.js:150scheduleResolveOrReject @ zone.js:460ZoneAwarePromise.then @ zone.js:536RuntimeCompiler._compileComponents @ runtime_compiler.js:157RuntimeCompiler._compileModuleAndComponents @ runtime_compiler.js:64RuntimeCompiler.compileModuleAsync @ runtime_compiler.js:55PlatformRef_._bootstrapModuleWithZone @ application_ref.js:303PlatformRef_.bootstrapModule @ application_ref.js:285(anonymous function) @ main.ts:12__webpack_require__ @ bootstrap 8ad11c6…:52(anonymous function) @ zone.js:1327__webpack_require__ @ bootstrap 8ad11c6…:52webpackJsonpCallback @ bootstrap 8ad11c6…:23(anonymous function) @ main.bundle.js:1
error_handler.js:51TypeError: Cannot read property 'name' of undefined
    at DataTableHeaderCell.get [as name] (index.umd.js:1263)
    at DebugAppView._View_DataTableHeader4.detectChangesInternal (DataTableHeader.ngfactory.js:820)
    at DebugAppView.AppView.detectChanges (view.js:272)
    at DebugAppView.detectChanges (view.js:377)
    at DebugAppView.AppView.detectContentChildrenChanges (view.js:290)
    at DebugAppView._View_DataTableHeader3.detectChangesInternal (DataTableHeader.ngfactory.js:620)
    at DebugAppView.AppView.detectChanges (view.js:272)
    at DebugAppView.detectChanges (view.js:377)
    at DebugAppView.AppView.detectContentChildrenChanges (view.js:290)
    at DebugAppView._View_DataTableHeader0.detectChangesInternal (DataTableHeader.ngfactory.js:131)ErrorHandler.handleError @ error_handler.js:51next @ application_ref.js:273schedulerFn @ async.js:82SafeSubscriber.__tryOrUnsub @ Subscriber.js:223SafeSubscriber.next @ Subscriber.js:172Subscriber._next @ Subscriber.js:125Subscriber.next @ Subscriber.js:89Subject.next @ Subject.js:55EventEmitter.emit @ async.js:74onError @ ng_zone.js:120onHandleError @ ng_zone_impl.js:64ZoneDelegate.handleError @ zone.js:207Zone.runTask @ zone.js:139drainMicroTaskQueue @ zone.js:368Promise.resolve (async)scheduleQueueDrain @ zone.js:341scheduleMicroTask @ zone.js:349ZoneDelegate.scheduleTask @ zone.js:219Zone.scheduleMicroTask @ zone.js:150scheduleResolveOrReject @ zone.js:460ZoneAwarePromise.then @ zone.js:536RuntimeCompiler._compileComponents @ runtime_compiler.js:157RuntimeCompiler._compileModuleAndComponents @ runtime_compiler.js:64RuntimeCompiler.compileModuleAsync @ runtime_compiler.js:55PlatformRef_._bootstrapModuleWithZone @ application_ref.js:303PlatformRef_.bootstrapModule @ application_ref.js:285(anonymous function) @ main.ts:12__webpack_require__ @ bootstrap 8ad11c6…:52(anonymous function) @ zone.js:1327__webpack_require__ @ bootstrap 8ad11c6…:52webpackJsonpCallback @ bootstrap 8ad11c6…:23(anonymous function) @ main.bundle.js:1
error_handler.js:54ERROR CONTEXT:ErrorHandler.handleError @ error_handler.js:54next @ application_ref.js:273schedulerFn @ async.js:82SafeSubscriber.__tryOrUnsub @ Subscriber.js:223SafeSubscriber.next @ Subscriber.js:172Subscriber._next @ Subscriber.js:125Subscriber.next @ Subscriber.js:89Subject.next @ Subject.js:55EventEmitter.emit @ async.js:74onError @ ng_zone.js:120onHandleError @ ng_zone_impl.js:64ZoneDelegate.handleError @ zone.js:207Zone.runTask @ zone.js:139drainMicroTaskQueue @ zone.js:368Promise.resolve (async)scheduleQueueDrain @ zone.js:341scheduleMicroTask @ zone.js:349ZoneDelegate.scheduleTask @ zone.js:219Zone.scheduleMicroTask @ zone.js:150scheduleResolveOrReject @ zone.js:460ZoneAwarePromise.then @ zone.js:536RuntimeCompiler._compileComponents @ runtime_compiler.js:157RuntimeCompiler._compileModuleAndComponents @ runtime_compiler.js:64RuntimeCompiler.compileModuleAsync @ runtime_compiler.js:55PlatformRef_._bootstrapModuleWithZone @ application_ref.js:303PlatformRef_.bootstrapModule @ application_ref.js:285(anonymous function) @ main.ts:12__webpack_require__ @ bootstrap 8ad11c6…:52(anonymous function) @ zone.js:1327__webpack_require__ @ bootstrap 8ad11c6…:52webpackJsonpCallback @ bootstrap 8ad11c6…:23(anonymous function) @ main.bundle.js:1
error_handler.js:55DebugContext {_view: _View_DataTableHeader4, _nodeIndex: 0, _tplRow: 29, _tplCol: 8}ErrorHandler.handleError @ error_handler.js:55next @ application_ref.js:273schedulerFn @ async.js:82SafeSubscriber.__tryOrUnsub @ Subscriber.js:223SafeSubscriber.next @ Subscriber.js:172Subscriber._next @ Subscriber.js:125Subscriber.next @ Subscriber.js:89Subject.next @ Subject.js:55EventEmitter.emit @ async.js:74onError @ ng_zone.js:120onHandleError @ ng_zone_impl.js:64ZoneDelegate.handleError @ zone.js:207Zone.runTask @ zone.js:139drainMicroTaskQueue @ zone.js:368Promise.resolve (async)scheduleQueueDrain @ zone.js:341scheduleMicroTask @ zone.js:349ZoneDelegate.scheduleTask @ zone.js:219Zone.scheduleMicroTask @ zone.js:150scheduleResolveOrReject @ zone.js:460ZoneAwarePromise.then @ zone.js:536RuntimeCompiler._compileComponents @ runtime_compiler.js:157RuntimeCompiler._compileModuleAndComponents @ runtime_compiler.js:64RuntimeCompiler.compileModuleAsync @ runtime_compiler.js:55PlatformRef_._bootstrapModuleWithZone @ application_ref.js:303PlatformRef_.bootstrapModule @ application_ref.js:285(anonymous function) @ main.ts:12__webpack_require__ @ bootstrap 8ad11c6…:52(anonymous function) @ zone.js:1327__webpack_require__ @ bootstrap 8ad11c6…:52webpackJsonpCallback @ bootstrap 8ad11c6…:23(anonymous function) @ main.bundle.js:1
zone.js:357ViewWrappedError {_nativeError: Error: Error in ./DataTableHeader class DataTableHeader - inline template:29:8 caused by: Cannot rea…, originalError: TypeError: Cannot read property 'name' of undefined
    at DataTableHeaderCell.get [as name] (http:/…, context: DebugContext}consoleError @ zone.js:357drainMicroTaskQueue @ zone.js:371Promise.resolve (async)scheduleQueueDrain @ zone.js:341scheduleMicroTask @ zone.js:349ZoneDelegate.scheduleTask @ zone.js:219Zone.scheduleMicroTask @ zone.js:150scheduleResolveOrReject @ zone.js:460ZoneAwarePromise.then @ zone.js:536RuntimeCompiler._compileComponents @ runtime_compiler.js:157RuntimeCompiler._compileModuleAndComponents @ runtime_compiler.js:64RuntimeCompiler.compileModuleAsync @ runtime_compiler.js:55PlatformRef_._bootstrapModuleWithZone @ application_ref.js:303PlatformRef_.bootstrapModule @ application_ref.js:285(anonymous function) @ main.ts:12__webpack_require__ @ bootstrap 8ad11c6…:52(anonymous function) @ zone.js:1327__webpack_require__ @ bootstrap 8ad11c6…:52webpackJsonpCallback @ bootstrap 8ad11c6…:23(anonymous function) @ main.bundle.js:1
zone.js:355Unhandled Promise rejection: Error in ./DataTableHeader class DataTableHeader - inline template:29:8 caused by: Cannot read property 'name' of undefined ; Zone: <root> ; Task: Promise.then ; Value: ViewWrappedError {_nativeError: Error: Error in ./DataTableHeader class DataTableHeader - inline template:29:8 caused by: Cannot rea…, originalError: TypeError: Cannot read property 'name' of undefined
    at DataTableHeaderCell.get [as name] (http:/…, context: DebugContext} TypeError: Cannot read property 'name' of undefined
    at DataTableHeaderCell.get [as name] (http://localhost:4200/main.bundle.js:46847:31)
    at DebugAppView._View_DataTableHeader4.detectChangesInternal (DataTableHeader.ngfactory.js:820:49)
    at DebugAppView.AppView.detectChanges (http://localhost:4200/main.bundle.js:60753:14)
    at DebugAppView.detectChanges (http://localhost:4200/main.bundle.js:60858:44)
    at DebugAppView.AppView.detectContentChildrenChanges (http://localhost:4200/main.bundle.js:60771:19)
    at DebugAppView._View_DataTableHeader3.detectChangesInternal (DataTableHeader.ngfactory.js:620:8)
    at DebugAppView.AppView.detectChanges (http://localhost:4200/main.bundle.js:60753:14)
    at DebugAppView.detectChanges (http://localhost:4200/main.bundle.js:60858:44)
    at DebugAppView.AppView.detectContentChildrenChanges (http://localhost:4200/main.bundle.js:60771:19)
    at DebugAppView._View_DataTableHeader0.detectChangesInternal (DataTableHeader.ngfactory.js:131:8)consoleError @ zone.js:355_loop_1 @ zone.js:382drainMicroTaskQueue @ zone.js:386Promise.resolve (async)scheduleQueueDrain @ zone.js:341scheduleMicroTask @ zone.js:349ZoneDelegate.scheduleTask @ zone.js:219Zone.scheduleMicroTask @ zone.js:150scheduleResolveOrReject @ zone.js:460ZoneAwarePromise.then @ zone.js:536RuntimeCompiler._compileComponents @ runtime_compiler.js:157RuntimeCompiler._compileModuleAndComponents @ runtime_compiler.js:64RuntimeCompiler.compileModuleAsync @ runtime_compiler.js:55PlatformRef_._bootstrapModuleWithZone @ application_ref.js:303PlatformRef_.bootstrapModule @ application_ref.js:285(anonymous function) @ main.ts:12__webpack_require__ @ bootstrap 8ad11c6…:52(anonymous function) @ zone.js:1327__webpack_require__ @ bootstrap 8ad11c6…:52webpackJsonpCallback @ bootstrap 8ad11c6…:23(anonymous function) @ main.bundle.js:1
zone.js:357Error: Uncaught (in promise): Error: Error in ./DataTableHeader class DataTableHeader - inline template:29:8 caused by: Cannot read property 'name' of undefined(…)

4.
**Expected behavior**
Expect the table is displayed

**Reproduction of the problem**

**What is the motivation / use case for changing the behavior?**
angular-cli integration

**Please tell us about your environment:**
angular-cli: 1.0.0-beta.16
node: 4.5.0
os: darwin x64

* **Table version:** 0.7.x
"angular2-data-table": "^0.9.0"

* **Angular version:** 2.0.x
    "@angular/common": "2.0.0",
    "@angular/compiler": "2.0.0",
    "@angular/core": "2.0.0",
    "@angular/forms": "2.0.0",
    "@angular/http": "2.0.0",
    "@angular/platform-browser": "2.0.0",
    "@angular/platform-browser-dynamic": "2.0.0",
    "@angular/router": "3.0.0",

* **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]
Chrome

* **Language:** [all | TypeScript X.X | ES6/7 | ES5]

**Comment by tscislo** [comment]
+1

**Comment by amcdnl** [comment]
Fixed in 0.9.2

**Comment by ipassynk** [comment]
Thanks. It works.
